tests/core/test_grid.py

```diff
@@ -1041,7 +1041,7 @@ def test_cast_uniform_to_structured():
 
 
 def test_cast_uniform_to_rectilinear():
-    grid = examples.load_uniform()
+    grid = pv.Wavelet()  # Uses an offset
```

Comment:
The tests should include these explicitly
```suggestion
    grid.offset = (1,2,3)
    grid.direction_matrix = np.diag((-1, -1, -1))
    grid.spacing = (1.1, 2.2, 3.3)
```